index.js

```diff
@@ -45,6 +45,7 @@ class Analytics {
     this.axiosInstance = axiosInstance
     this.timeout = options.timeout || false
     this.flushAt = Math.max(options.flushAt, 1) || 20
+    this.maxQueueSize = options.maxQueueSize || 1024 * 32 // defaults to 32kb
```

Comment:
shall we use larger queue size based on the feedback we got on the other libraries?

Reply:
We should. I'll make the change